esignet-service/.env.example

```
@@ -35,12 +35,72 @@ GATE_CLIENT_PORT=8080
 # BASE_URL=http://127.0.0.1:8080
 
+# ── PostgreSQL (client management persistence) ────────────────────────────────
+# Option A: full DSN (takes precedence over individual vars)
+# POSTGRES_URL=postgres://esignet:secret@localhost:5432/esignet?sslmode=disable
+
+# Option B: individual connection params
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=mosip_esignet
+POSTGRES_USER=esignet
+POSTGRES_PASSWORD=secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_PORT key

(UnorderedKey)